src/overlays/actors/ovl_En_Wiz/z_en_wiz.h

```diff
@@ -2,22 +2,78 @@
 #define Z_EN_WIZ_H
 
 #include "global.h"
+#include "objects/object_wiz/object_wiz.h"
+
+#define EN_WIZ_GET_SWITCHFLAG(thisx) ((thisx)->params & 0x7F)
+#define EN_WIZ_GET_TYPE(thisx) (((thisx)->params >> 8) & 0xFF)
+
+typedef enum {
+    /* 0 */ EN_WIZ_TYPE_FIRE,
+    /* 1 */ EN_WIZ_TYPE_ICE,
+    /* 2 */ EN_WIZ_TYPE_FIRE_NO_MINI_BOSS_BGM,
```

Comment:
I think we can probably shorten this a bit, maybe put `MINI_BOSS` in the other two? And note the bgm thing here if you want to keep it.

Reply:
I'm not sure what exactly you mean here. They're *all* mini-bosses; what sets type 2 apart is that it doesn't play the mini-boss BGM. I'm not sure how to make it shorter without just completely eliminating the only thing that makes it different.

Reply:
Maybe just `EN_WIZ_TYPE_FIRE_NO_BGM`? With a comment saying it doesn't request the Mini-Boss Bgm?

Reply:
I wouldn't necessarily say they're all minibosses, Stalfos, Lizalfos and Dinofos do the same separation too, in both games. But yeah, since the only difference seems to be the bgm or lack thereof, engineer's suggestion's probably better here.

Reply:
Yeah, I guess in my head, I figured "they lock you in a room until you beat this guy; he has to be a miniboss," which ignores that they lock you in rooms with regular enemies all the time. I'll go with what engineer suggested.